mitodl/open-discussions
# add about field to channel model

#### Pre-Flight checklist



#### What are the relevant tickets?

part of #1778 

#### What's this PR do?

this adds a `JSONField` called `about` to the `Channel` object, and then also adds it to the serializer for the `Channel`.

#### How should this be manually tested?

Make sure that you can set the value of the `about` field if you use the DRF view for the API (`/api/v0/channels/${channelName}`). It should only let you set the value to be a list (what will be using for the ckeditor document) and it should give you an error if you try to set it to be something like `{foo: "bar"}` (but `[{foo: "bar" }]` should be good).
